webapp/src/containers/Rhs/projectList/index.tsx

```diff
@@ -78,13 +82,13 @@ const ProjectList = () => {
                     title='Confirm Project Unlink'
                 />
             }
-            {isLoading && <LinearLoader/> }
+            {isLoading && <LinearLoader/>}
             {
                 isSuccess && (
-                    projectsList?.length > 0 ?
+                    sortedProjectList.length > 0 ?
                         <>
                             {
-                                projectsList.map((item: ProjectDetails) => (
+                                sortedProjectList.map((item: ProjectDetails) => (
```

Comment:
We can improve the performance by memoizing the `sortedProjectList`. For now, it'll be re-created on each render and because of that, the whole list of projects will be re-rendered on each render no matter whether the projects are changed or not.